omnigibson/systems/macro_particle_system.py

```diff
@@ -30,6 +30,8 @@ class MacroParticleSystem(BaseSystem):
     # Note that this object is NOT part of the actual particle system itself!
     _particle_object = None
 
+    _particle_template = None
```

Comment:
Do we need to track these separately? Is _particle_object not always _particle_template.visual_meshes[0]? If so, can we replace it with a property or something so that the duplication is removed?